clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/manifest/KubernetesDeployManifestOperation.java

```diff
@@ -218,6 +158,85 @@ public OperationResult operate(List<OperationResult> _unused) {
     return result;
   }
 
+  private List<KubernetesManifest> bindArtifacts(
```

Comment:
It looks like the order of operations has changed from bind then version to version then bind (in `bindArtifacts`). At the same time we still try to version (line 102). Was it done on purpose? Looking up the version is a costly operation.